src/components/ContactForm/ContactBtn/ContactBtn.tsx

```diff
@@ -0,0 +1,44 @@
+'use client'
+import React, { useState } from 'react'
+import { Button } from '@/components/Button/Button'
+
+import ContactFormModal from '../ContactFormModal/ContactFormModal'
+import { ProfileModel } from '@/data/frontend/profile/types'
+import { useModal } from '@/contexts/ModalContext'
+import ContactSuccessModal from '../ContactSuccessModal/ContactSuccessModal'
+
+const ContactBtn = ({ userProfile }: { userProfile: ProfileModel }) => {
+  const [showFormModal, setShowFormModal] = useState<boolean>(false)
```

Comment:
**nitpick:** I think we should be able to do this with `useModal` hook instead of duplicating this logic

Reply:
Yeah, you're right! Done.